pkg/services/searchV2/extract/testdata/empty-datasource-variable-info.json

```diff
@@ -7,8 +7,8 @@
   ],
   "datasource": [
     {
-      "uid": "default.uid",
-      "type": "default.type"
+      "uid": "${sqllite}",
```

Comment:
the UI's default behavior is to go with the datasource if a ds from variable can't be found, so I replicated this here. It's definitely not great, but I was looking at it mainly from the perspective of permissions.